vignettes/c-using-rix-to-build-project-specific-environments.Rmd

```diff
@@ -141,7 +141,7 @@ project is called `housing`, you could create this script and execute it to
 start your project:
 
 ```
-!#/bin/bash
+#!/bin/bash
```

Comment:
This needs to go in  `dev/c-building_envs_with_rix.Rmd` because we're using {fusen} to develop the package (see CONTRIBUTING.md)

Reply:
Well noted. I'll leave it to your care.